Optimisation

GabrielRavier opened the pull request:
I optimised a bunch of shit and corrected a few bugs

Leo40Git commented:
Instead of making empty labels everywhere, could you make it so that saveSpace simply doesn't include the unused objects/maps?

GabrielRavier commented:
I can't do that for one simple reason : Most of those labels are actually referenced (like for exemple the objects I'm removing the code for are referenced in the object table), but never jumped to. If I removed them, I would get an error telling me the label is referenced but doesn't exist. If I removed those references, I would have to remake all the levels so that the entity numbers are aligned.

Leo40Git commented:
just make their entries point to NullObject
